packages/koenig-lexical/src/components/ui/cards/CallToActionCard.jsx

```diff
@@ -378,5 +382,6 @@ CallToActionCard.propTypes = {
     visibilityOptions: PropTypes.array,
     toggleVisibility: PropTypes.func,
     imageUploadHandler: PropTypes.func,
-    imageDragHandler: PropTypes.object
+    imageDragHandler: PropTypes.object,
+    imageUploader: PropTypes.func
```

Comment:
I think this is an `object` rather than a `func`? Worth keeping dev tools open and watching the console whilst developing, this sort of mismatch usually creates error logs